assets/app/view/game/corporation.rb

```diff
@@ -348,7 +348,8 @@ def render_operating_order
 
         round = @game.round
         if (n = @game.round.entities.find_index(@corporation))
-          span_class = '.bold' if n >= round.entities.find_index(round.current_entity)
+          m = round.entities.find_index(round.current_entity)
```

Comment:
index i believ is the same

Reply:
Yes, index is just an alias to find_index. I changed to use index in both lines now as it is shorter.